jonas-periodic-table/style.css

```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&display=swap');
+
+* {
+    margin: 0px;
+    font-family: 'Bree Serif', serif;
+}
+
+.site-container {
+
+    width: fit-content;
+    height: fit-content;
+    margin-top: 40px;
+    margin-left: 100px;
+}
+
+.periodic-row {
+display: flex;
+flex-direction: row;
+}
+
+.cell {
+    width: 95px;
+    height: 85px;
+    justify-content: space-between;
+    margin-top: 1px;
+    margin-bottom: 1px;
+    box-sizing: border-box;
+    box-shadow: 1px 1px 1px rgba(255,255,255,0.5);
+    border: 1px solid rgba(0,0,0,0.05);
+    text-align: center;
+    position: sticky;
+    align-self: flex-start;
+}
+
+.cell:hover{
+    color: #ffffff;
+  }
+
+.symbol {
+    font-size: 30px;
+}
+
+.at_num {
+    font-size: 10px;
+    text-align: right;
+}
+
+.at_details {
+    font-size: 12px;
+}
+
+.cell2 {
+    width: 95px;
+    height: 85px;
+    justify-content: space-between;
+    margin-top: 1px;
+    margin-bottom: 1px;
+    box-sizing: border-box;
+    text-align: center;
+    position: sticky;
+    align-self: flex-start;
+}
+
+.cell:nth-child(-n+2) .element {
+    background-color: rgb(173, 119, 19);
+  }
+
+  .cell:nth-child(1) .element {
+    background-color: rgb(218, 91, 45);
+  }
+  .periodic-row:nth-child(4) .cell:nth-child(n+3) .element,
+  .periodic-row:nth-child(5) .cell:nth-child(n+3) .element,
+  .periodic-row:nth-child(6) .cell:nth-child(n+4) .element,
+  .periodic-row:nth-child(7) .cell:nth-child(n+4) .element {
+    background-color: rgb(146, 65, 72);
+  }
+
+  .periodic-row:nth-child(2) .cell:nth-child(n+13) .element,
+  .periodic-row:nth-child(3) .cell:nth-child(n+13) .element,
+  .periodic-row:nth-child(4) .cell:nth-child(n+13) .element,
+  .periodic-row:nth-child(5) .cell:nth-child(n+13) .element,
+  .periodic-row:nth-child(6) .cell:nth-child(n+13) .element,
+  .periodic-row:nth-child(7) .cell:nth-child(n+13) .element {
+    background-color: rgb(161, 161, 156);
+  }
+
+  .periodic-row:nth-child(2) .cell:nth-child(n+14) .element,
+  .periodic-row:nth-child(3) .cell:nth-child(n+15) .element,
+  .periodic-row:nth-child(4) .cell:nth-child(n+16) .element,
+  .periodic-row:nth-child(5) .cell:nth-child(n+17) .element {
+    background-color: rgb(146, 146, 15);
```

Comment:
I think a bit better way to do it is to use  .blue / .red / .yellow modifier classes, but it's not very important